lib/msgraph/managed_device.go

```diff
@@ -37,7 +37,6 @@ type ManagedDevice struct {
 	// certificateReset, notRegisteredPendingEnrollment, unknown.
 	DeviceRegistrationState string `json:"deviceRegistrationState"`
 	SerialNumber            string `json:"serialNumber"`
-	Model                   string `json:"model"`
```

Comment:
Maybe leave a comment somewhere mentioning we don't sync Model because of the Intune/tsh mismatch?

Could be here or on e/, whatever you think is best.